sched_boundary/sched_boundary.py

```diff
@@ -99,6 +99,11 @@ def fix(decl, str):
             str = bugfix(decl, str)
         return str
 
+    @staticmethod
+    def handle3dots(decl, signature):
+        if decl.str_decl.find("...") >= 0:
+            signature["params"] += ", ..."
```

Comment:
try to use replace().

Reply:
str_decl.find("...") >= 0 means there is "..." in str_decl. 
using str_del.replace("...",  ", ...") to keep the same as other code.

Reply:
decl.str_decl is pure text from [[origin source kernel code]]. We can use it to check whether there is "..." but cannot directly use it.
The information extracted by gcc is stored in somewhere else. (e.g., return type is in decl.result, and params are in decl.arguments)

Here signature["params"] is generated by decl.arguments, and decl.arguments miss "..."

Reply:
> decl.str_decl is pure text from [[origin kernel source code]]. 
Oh... there's a little wrong...
decl.str_decl is not 100% from origin kernel source code.

e.g., "enum" will miss, and "va_list" will become "struct  *" in decl.str_decl. (While "..." remains in it.)
So we could not use decl.str_decl when exporting to c files.